doc/release-process.md

```diff
@@ -12,6 +12,26 @@ It is best if the maintainers are present during the release, so they can help e
 
 This process also assumes that there will be no minor releases for old major releases.
 
+## Sanity Checks
+Perform these checks before creating a release:
+
+1. Ensure `make distcheck` doesn't fail.
+```shell
+make distcheck
+```
+2. Check installation with autotools:
+```shell
+dir=$(mktemp -d)
+./configure --prefix=$dir && make install && ls -l $dir/include $dir/lib
+gcc -o ecdsa examples/ecdsa.c -I $dir/include -L $dir/lib/ -l secp256k1 && ./ecdsa
+```
+3. Check installation with CMake:
+```shell
+dir=$(mktemp -d)
+mkdir build && cmake -B build -DCMAKE_INSTALL_PREFIX=$dir && cmake --build build --target install && ls -l $dir/include $dir/lib*
+gcc -o ecdsa examples/ecdsa.c -I $dir/include -L $dir/lib*/ -l secp256k1 && ./ecdsa
```

Comment:
The last command, `./ecdsa`, needs the `LD_LIBRARY_PATH` variable to specify the path to the `libsecp256k1.so.2` library.

Reply:
Right, that's the alternative to static linking I suggested above.

Maybe all of this sanity-checking should go to a small script that then can also runs on CI then?

Reply:
https://github.com/bitcoin-core/secp256k1/pull/1310#discussion_r1192956630